recipes/numcpp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "NumCpp" %}
+{% set version = "2.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dpilger26/{{ name }}/archive/refs/tags/Version_{{ version }}.tar.gz
+  sha256: f462ecd27126e6057b31fa38f1f72cef2c4223c9d848515412970714a5bb6d16
+
+build:
+  number: 0
+  skip: True  # [not unix]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=3.14
+  host:
+    - boost
```

Comment:
`boost` is the Python package. You probably only want the C++ part here?

Reply:
thanks, fixed